Domain/Sources/Domain/UseCase/UserUseCase.swift

```diff
@@ -31,12 +31,13 @@ public struct UserUseCase {
             }
         }
     }
-    public func login(loginInfo localLoginInfo: (userName:String,passWord:String), completion: @escaping (Token) -> Void) {
+    public func login(loginInfo localLoginInfo: (userName:String,passWord:String), completion: @escaping (String?)-> Void) {
```

Comment:
why changed here ?

Reply:
Because in the view model i want to check if it returns a string or not only, if it returns a string so he is a registered user, Hassan who made this part was in a meeting with me and we have done this.